### PR TITLE
Feature/integrate react query

### PR DESCRIPTION
**Pull request by vinayak-4**

### CODE REVIEW

**Review by shivam910 — COMMENTED** [review]
Can we add some dummy image, if no image  is there 
Screenshot_2023-02-05-21-25-39-86_40deb401b9ffe8e1df2f1cc5ba480b12

**Review by shivam910 — COMMENTED** [review]
Please add this in env file

**Review by shivams10 — COMMENTED** [review]
Can we add some dummy image when we get no image ?
Screenshot 2023-02-05 at 9 36 16 PM

**Review by shivams10 — COMMENTED** [review]
Please add this in env file
Screenshot 2023-02-05 at 9 38 32 PM

File: src/pages/Browse/index.jsx [hunk 1]
```diff
@@ -1,96 +1,58 @@
...
         name,
+        slug,
       });
     }
   }
   function handleClick() {
-    refetchGames();
-    setGenre((previous) => ({
+    // refetchGames();
+    setGenre(() => ({
```
Comment (Himanshur25) [inline src/pages/Browse/index.jsx:47]: Remove Comments.

**Pull request merged**